server/model/newExamResultsSubscriber.go

```diff
@@ -0,0 +1,13 @@
+package model
+
+import (
+	"database/sql"
+	"time"
+)
+
+type NewExamResultsSubscriber struct {
+	CallbackUrl    string         `gorm:"primary_key" json:"callbackUrl"`
+	ApiKey         sql.NullString `json:"-"`
```

Comment:
We switched to only using `null` instead of mixing the two in recent PRs

```suggestion
	ApiKey         null.String `json:"-"`
```